### PR TITLE
Added Find-Files functionality as common module

### PR DESCRIPTION
**Pull request by nigurr**
* Find files from any folder
* Supports legacy find patterns `[+:C:\\a\\*.dll,-:C:\\b\\*.dll]`
* UTs are added

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
rename to find-files-legacy since this is OSS and we want to encourage folks to use the lib and minimatch.

File: Tests/L0/Common-FindFiles-Legacy/_suite.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    });
+
+    it('Search recursively with include files', (done) => {
+        let test = ff.findFiles(['+:' + path.join(data, '**\\*.log')]);
+        assert(test.length === 4);
+        done();
+    });
+
+    it('Search recursively with exclude files', (done) => {
+        let test = ff.findFiles([path.join(data, '**\\*'), '-:' + path.join(data, '**\\*.log')]);
```
Comment (acesiddhu) [inline Tests/L0/Common-FindFiles-Legacy/_suite.ts]: where is the log file

**Comment by acesiddhu** [comment]
:shipit:

File: Tests/L0/Common-FindFiles-Legacy/_suite.ts (hunk 1 above)
Comment (nigurr) [inline Tests/L0/Common-FindFiles-Legacy/_suite.ts]: added

**Pull request merged**